content/news/playtest-20150801.md

```diff
@@ -0,0 +1,58 @@
+We are pleased to announce the second playtest on the road to our next stable release.  We have now entered feature freeze phase, which means that we have stopped adding new features and instead focus on fixing any bugs that you might find.  That means that we need your help with testing, testing and some more testing!  Please report any bugs and problems you encounter [on our bug tracker](http://bugs.openra.net). Thank you for helping to improve OpenRA!
+
+A number of new features, fixes, and performance improvements make this a significant improvement over the last playtest and the June release.
+
+We are happy to announce that we have finally addressed a few long-standing performance issues that caused significant lag spikes in the mid to late game.
+As part of this work, the AI has become a lot smarter about base building, uses support powers more reliably, and harvesters are less likely to become stuck.
+
+Music playback has received quite a lot of changes. Map authors now have much greater control over music, including the ability to specify default (starting) music and custom victory and defeat tracks.
+
+Other changes include:
+
+* Spies in the Red Alert mod have received a major overhaul, making them much more useful.
+* Paratroopers can now be selected and given orders before they land.
+* Two new single-player missions have been added, Soviets 04a and 04b.
+* Single-player missions have been polished, and Allies 05 has received a new ultra-hard difficulty.
+* Mission replays can again be viewed through the replay browser.
```

Comment:
Is this a new feature or a fix? If it is new, you should probably remove again; if not, move it vv